plugins/org.jkiss.dbeaver.ui/src/org/jkiss/dbeaver/ui/dialogs/IWizardPageNavigable.java

```diff
@@ -31,5 +31,12 @@ public interface IWizardPageNavigable {
      * Non-applicable pages are not visible in navigation tree.
      */
     boolean isPageApplicable();
-    
+
+    /**
+     * Indicates if page is the last step in the wizard.
+     * Set to false by default, needs to be overridden by implementation if necessary.
```

Comment:
Looks like a generated comment, the first sentence is enough

Reply:
Seems okay to me

Reply:
I'm fine with this comment too.